xapian-letor/include/xapian-letor/feature.h

```diff
@@ -104,35 +106,50 @@ class XAPIAN_VISIBILITY_DEFAULT Feature {
     /// Virtual destructor because we have virtual methods.
     virtual ~Feature();
 
-    /// Specify the database to use for feature building. This will be used by FeatureList::Internal class.
+    /** Specify the database to use for feature building.
+     *  This will be used by FeatureList::Internal class.
+     */
```

Comment:
Thanks for reformatting these, but you've missed the part about adding an empty line between the brief description and the detailed description.  Here's an example of a doc comment with brief description, detailed description and parameter and return value descriptions showing the standard formatting we use:

https://trac.xapian.org/browser/git/xapian-core/include/xapian/queryparser.h#L197